physical/azure/azure.go

```diff
@@ -195,7 +229,7 @@ func (a *AzureBackend) Get(ctx context.Context, key string) (*physical.Entry, er
 
 	reader := res.Body(azblob.RetryReaderOptions{})
 
-	defer reader.Close()
+	defer func() { _ = reader.Close() }()
```

Comment:
I'm taking a guess that this was wrapped to avoid the warning from the linter, but we should be fine calling defer reader.Closer() directly in here since we're not acting on the error.

Reply:
Correct, this only makes it explicit that we're not acting on the error and suppresses the warning

Reply:
The anonymous func doesn't seem necessary, can we revert this?

Reply:
It's fine to call `Close` directly and not handle the error explicitly since the error can/is being ignored. We do this [throughout vault's codebase](https://github.com/hashicorp/vault/blob/6f1749a88efe573d9654fbc943a5cff9fbda6228/api/auth_token.go#L27), and is also an acceptable practice as shown in the stdlib's [http package](https://golang.org/pkg/net/http/).

Reply:
Okay I want to adhere to the style of the repo so I'll change it back

Reply:
done!